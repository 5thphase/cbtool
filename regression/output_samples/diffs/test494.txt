Background operation execution success. Operation "vm-attach" will be processed asynchronously, through the command ""/path/to/tool/lib/operations//../../cbact" --procid=TEST_msilva --osp=port:xxxx,experiment_inst:TEST_msilva,host:xxx.xxx.xxx.xxx ,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:xxxx,password:cbpwdZ9,timeout:240,host:xxx.xxx.xxx.xxx ,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,tinyvm,auto,empty,iron32,pause_provision_complete,empty=empty --operation=vm-attach --cn=TESTCLOUD --uuid=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx --daemon". The process id is xxxx.
 status: Counter "VM ARRIVING" equals 1 after 0.380064964294 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 1.38006496429 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 2.38006496429 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 3.38006496429 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 4.38006496429 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 5.38006496429 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 6.38006496429 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 7.38006496429 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 8.38006496429 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 9.38006496429 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 0 after 10.3800649643 seconds (the counter is assumed to be decreasing).
Waited 10.3800649643 seconds until "VMs ARRIVING" was equal to 0.
|Redis Version |3.0.6 
|VMs APPLICATION ERRORS |5 
