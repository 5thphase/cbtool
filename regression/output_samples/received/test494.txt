Background operation execution success. Operation "vm-attach" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.1.250,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,timeout:240,host:172.16.1.250,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,tinyvm,auto,empty,iron32,pause_provision_complete,empty=empty --operation=vm-attach --cn=TESTCLOUD --uuid=6E33D662-9674-5EAD-831A-203B559B0C71 --daemon". The process id is 3202.

Going to unconditionally wait for 8s (8 seconds). The command line interface will be blocked during the waiting.
Waited 1 seconds... (12.5%)
Waited 2 seconds... (25.0%)
Waited 3 seconds... (37.5%)
Waited 4 seconds... (50.0%)
Waited 5 seconds... (62.5%)
Waited 6 seconds... (75.0%)
Waited 7 seconds... (87.5%)
Waited 8 seconds... (100.0%)
Waited for 8 seconds.
Message "vm_12;continue;something" published on channel "staging" (object "VM" ).
Going to wait until the value on counter "VM ARRIVING" is equal to 0 (currently it is equal to 1) waiting 1.0 seconds between samples. The counter is assumed to be decreasing.
 status: Counter "VM ARRIVING" equals 1 after 1 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 2 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 3 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 4 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 5 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 6 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 0 after 7 seconds (the counter is assumed to be decreasing).
Waited 7 seconds until "VMs ARRIVING" was equal to 0.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |238.219.171.43  |simhostc1                  |SUT                 |none    |none       |5C3901F8-BB8C-5D41-AFA5-8507C3FB1C4A
|vm_3          |tinyvm              |pico32      |95.156.75.163   |simhostb1                  |SUT                 |none    |none       |764DCC85-E8B4-535C-892F-21EAA7EB7475
|vm_4          |db2                 |copper32    |142.190.173.104 |simhosta1                  |SUT                 |none    |none       |9B1B97E3-95E9-5B28-960B-4F71F9119D35
|vm_5          |netclient           |platinum64  |59.115.15.24    |simhostc1                  |SUT                 |none    |none       |84D06D13-F388-55E9-987A-389725200A3F
|vm_6          |tinyvm              |pico32      |17.168.188.192  |simhostc1                  |SUT                 |none    |none       |37A1C15B-7C48-5592-9361-7A372C1177CC
|vm_7          |tinyvm              |pico32      |148.235.207.74  |simhostb1                  |SUT                 |none    |none       |AEC87C35-3F3B-542C-ABB0-962015B573E6
|vm_8          |tinyvm              |pico32      |203.20.69.18    |simhostc1                  |SUT                 |none    |none       |CDFD3CF9-DAAE-5E08-950E-49A139A8FDFC
|vm_9          |db2                 |platinum64  |218.25.48.46    |simhosta1                  |SUT                 |none    |none       |13FE56A6-3A9A-5DC2-AC8C-087D336BFCB0
|vm_10         |netclient           |iron32      |96.171.238.129  |simhosta2                  |SUT                 |none    |none       |64881098-FFF1-5453-8332-4C011C22E898
|vm_11         |fen_hpc             |platinum64  |45.168.211.108  |simhostd1                  |LG                  |none    |none       |A60D99ED-6BAB-5D66-91D5-A5209225184E
|vm_12         |tinyvm              |iron32      |172.201.240.253 |simhostc1                  |SUT                 |none    |none       |6E33D662-9674-5EAD-831A-203B559B0C71

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |1.50M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |1793059                       
|Total Connections Received                           |5127                          
|Total Commands Processed                             |819685                        
|Number of Keys                                       |375                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.6                         
|Storage Size                                         |1511424                       
|Data Size                                            |473616                        
|Index Size                                           |376096                        
|Average Object Size                                  |518.746987952                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |11                            
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |11                            
|VMs ARRIVED                                          |12                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |1                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |101                           

 ---------------- VMC-WIDE COUNTERS ----------------
|C3B836AF-6AAE-5401-84E8-B00701C39D77 (simzone_d) VM RESERVATIONS|1                             
|2F03C41C-37EA-529D-873F-A59AD9D097ED (simzone_b) VM RESERVATIONS|2                             
|85B1573F-AFCB-5D91-861C-A9290E0BC814 (simzone_a) VM RESERVATIONS|3                             
|9EA9AC96-B5DE-50B4-93B7-F0933B0AEB65 (simzone_c) VM RESERVATIONS|5                             

