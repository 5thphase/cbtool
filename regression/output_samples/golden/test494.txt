Background operation execution success. Operation "vm-attach" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.0.4,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,password:cbpwdZ9,timeout:240,host:172.16.0.4,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,tinyvm,auto,empty,iron32,pause_provision_complete,empty=empty --operation=vm-attach --cn=TESTCLOUD --uuid=3F8128E8-A398-5355-BBA1-9CB823AB4103 --daemon". The process id is 9607.

Going to unconditionally wait for 8s (8 seconds). The command line interface will be blocked during the waiting.
Waited 1 seconds... (12.5%)
Waited 2 seconds... (25.0%)
Waited 3 seconds... (37.5%)
Waited 5 seconds... (62.5%)
Waited 6 seconds... (75.0%)
Waited 7 seconds... (87.5%)
Waited 8 seconds... (100.0%)
Waited for 8 seconds.
Message "vm_12;continue;something" published on channel "staging" (object "VM").
Going to wait until the value on counter "VM ARRIVING" is equal to 0 (currently it is equal to 1) waiting 1.0 seconds between samples, for a maximum of 36000.0 seconds. The counter is assumed to be decreasing.
 status: Counter "VM ARRIVING" equals 1 after 0.937482118607 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 1.93748211861 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 2.93748211861 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 3.93748211861 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 4.93748211861 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 5.93748211861 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 0 after 6.93748211861 seconds (the counter is assumed to be decreasing).
Waited 6.93748211861 seconds until "VMs ARRIVING" was equal to 0.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool      |netname    |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |229.34.189.54   |simhostc12                 |SUT           |private    |none    |none       |FE0440AA-ABE8-54D3-BB2B-5F260B36067D
|vm_3          |tinyvm              |pico32      |193.123.238.143 |simhostb1                  |SUT           |private    |none    |none       |A7FD0203-E5C5-5070-8DDE-C87A3E4E302F
|vm_4          |db2                 |copper32    |130.168.194.254 |simhosta3                  |SUT           |private    |none    |none       |325854B7-33E2-5B6E-9B69-57EDC6FE865A
|vm_5          |netclient           |platinum64  |198.62.168.124  |simhostc2                  |SUT           |private    |none    |none       |716FA71C-EA4E-50CD-B018-61526F4B0B65
|vm_6          |tinyvm              |pico32      |106.9.243.243   |simhostb5                  |SUT           |private    |none    |none       |8B038382-CC58-55AC-B970-10F058A9CF4E
|vm_7          |tinyvm              |pico32      |196.241.129.46  |simhostc9                  |SUT           |private    |none    |none       |60A5B19D-E762-52E4-A8E6-C95EE331D498
|vm_8          |tinyvm              |pico32      |34.102.144.192  |simhostc3                  |SUT           |private    |none    |none       |B8414E3C-252C-584F-BAF1-D72E5FCA6C7E
|vm_9          |db2                 |platinum64  |16.50.116.206   |simhostb13                 |SUT           |private    |none    |none       |1A07BF13-EBA1-57D3-96E8-6D39FAE13839
|vm_10         |netclient           |iron32      |236.198.248.190 |simhosta2                  |SUT           |private    |none    |none       |74A61B10-7A62-5681-9998-8D2042DABF0E
|vm_11         |fen_hpc             |platinum64  |118.239.10.35   |simhostd13                 |LG            |private    |none    |none       |E07D13B0-041B-5177-8C0A-C56B54D498D8
|vm_12         |tinyvm              |iron32      |82.46.121.56    |simhosta11                 |SUT           |private    |none    |none       |3F8128E8-A398-5355-BBA1-9CB823AB4103

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |19.13M                        
|Redis Version                                        |2.8.6                         
|Uptime (in seconds)                                  |14435898                      
|Total Connections Received                           |614757                        
|Total Commands Processed                             |58724230                      
|Number of Keys                                       |798                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |28                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |11                            
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |11                            
|VMs ISSUED                                           |12                            
|VMs ARRIVED                                          |12                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |1                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |6                             
|AI RESERVATIONS                                      |0                             
|AIs ISSUED                                           |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ISSUED                                        |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |221                           

 ---------------- VMC-WIDE COUNTERS ----------------
|2C0CBD0C-C4AB-589F-B1DC-473FE1792EEA (simzone_b) VM RESERVATIONS|3                             
|09BE1FD2-5286-5B6E-AA11-D68D33341047 (simzone_a) VM RESERVATIONS|4                             
|95B3CBBD-180B-59D1-9048-B3EAC63E092B (simzone_d) VM RESERVATIONS|1                             
|4DFFE7AE-4E70-5DA7-82EF-D0BC8960A44B (simzone_c) VM RESERVATIONS|3                             

